test-data/unit/plugins/config_data.py

```diff
@@ -0,0 +1,18 @@
+import os
+import json
+
+from typing import Any
+
+from mypy.plugin import Plugin
+
+
+class ConfigDataPlugin(Plugin):
+    def get_config_data(self, id: str, path: str) -> Any:
+        path = os.path.join('tmp/test.json')
```

Comment:
This looks suspicious.

Reply:
Yeah but a lot about our test framework is suspicious :P

Reply:
OK, I give up, how does this work on Windows? Is this some kind of `pytest` magic? I would not use it and rather write `os.path.join('tmp', 'test.json')`

Reply:
Windows allows `/` in paths as a separator. Problems with Windows paths usually happen when you read a path value, like `path.split('/')` (since it likely uses `\`) or comparing output to expected values (again because the paths usually are rendered with `\`).

Reply:
Interesting, TIL.